### ISSUE TITLE
Failed to minify code

### ISSUE THREAD

**Issue by almaron** [xpl/es7-object-polyfill#2]
I've used `create-react-app` to create and set up my app. I then added `es7-object-polyfill` and required it in `src/index.js` right after all the imports.

```
import React from 'react'
import { render } from 'react-dom'
import { Provider } from 'react-redux'
import { ConnectedRouter } from 'react-router-redux'
import store, { history } from './redux/store/configure-store'
import AppRouter from './AppRouter'
import { unregister } from './registerServiceWorker';
import './styles/index.css'

require('es7-object-polyfill')

const target = document.querySelector('#root')

render(
    <Provider store={store}>
        <ConnectedRouter history={history}>
            <AppRouter />
        </ConnectedRouter>
    </Provider>,
    target
);

unregister();
```

When I try to run `yarn build` I get this error message.
```
Failed to compile.

Failed to minify the code from this file: 

        ./node_modules/es7-object-polyfill/es7-object-polyfill.js:11 

Read more here: http://bit.ly/2tRViJ9

```

**Comment by xpl** [comment]
@almaron Hello! May I ask why in 2019 you would need polyfilling `Object.values` or `Object.entries`? It has been supported for a long time in all major browsers natively:

<img width="1037" alt="Screen Shot 2019-03-20 at 13 28 11" src="https://user-images.githubusercontent.com/1707/54677485-0b264f80-4b14-11e9-9905-68b60a64260e.png">

**Comment by vitormv** [comment]
@xpl Samsung Internet only supports it after version 6.0. Version 5 was released in May 2017, it's not so old. Version 5 shipped with the Galaxy S8 for example. If user never updates the apps from the Galaxy Store, they will get the error. And that is only one browser/example, im sure there are many more use cases for being able to transpile the code.

Source: https://en.wikipedia.org/wiki/Samsung_Internet_for_Android

**Comment by vitormv** [comment]
The use case here being, if we need to import this package, that means we are dealing with old browsers that need to be supported (its sad i know, but we have to do it). And usually packages inside `node_modules/*` are never compiled because they are expected to come already transpiled to older versions. So here i would have two choices:
1. manually change webpack to explicitly compile this one folder inside `node_modules`
2. or just copy/paste the polyfills from this library into my own `src/polyfill.js` file, so it gets compiled along with the application :)

For now, went with the 2nd, but would be nice to just import this polyfill and forget about it without any extra tweaks :)

**Comment by xpl** [comment]
@vitormv Thanks for the clarification!

Do you have any suggestion why does it fail, btw? I know that the minifier in popular frameworks fails when it sees some non-ES5 syntax, like classes (what a shame in 2019), but AFAIR there is none of such...

**Comment by xpl** [comment]
Fixed in ed3a161fe12c7db618e85235772c03f42ef1a229

Let me know if the issue with minification persists...
